projects/l2_pooling/multi_column_convergence.py

```diff
@@ -94,6 +94,42 @@ def averageConvergencePoint(inferenceStats, prefix, minOverlap, maxOverlap,
   return convergenceSum/len(inferenceStats)
 
 
+
+def averageConvergencePointNew(numActiveL2cells, minOverlap, maxOverlap):
```

Comment:
@lscheinkman We should refactor this to use the common function you created in https://github.com/numenta/htmresearch/pull/793